src/metal/metal-device.cpp

```
@@ -403,34 +403,73 @@ Result DeviceImpl::getAccelerationStructureSizes(
     return SLANG_OK;
 }
 
+uint32_t DeviceImpl::registerAccelerationStructure(MTL::AccelerationStructure* accelerationStructure)
+{
+    SLANG_RHI_ASSERT(accelerationStructure);
+
+    uint32_t index = 0;
+    if (!m_accelerationStructures.freeList.empty())
+    {
+        index = m_accelerationStructures.freeList.back();
```

Cppcheck (2.20.0)

[style] The function 'setDesc' is never used.

(unusedFunction)